skyvern/library/embedded_server_factory.py

```
@@ -8,12 +10,76 @@
+    if llm_key:
+        settings.LLM_KEY = llm_key
+    if secondary_llm_key:
+        settings.SECONDARY_LLM_KEY = secondary_llm_key
+
+    # Apply custom settings overrides
+    if settings_overrides:
+        for key, value in settings_overrides.items():
+            if hasattr(settings, key):
+                setattr(settings, key, value)
+            else:
+                raise ValueError(f"Invalid setting: {key}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)